data/projects/optimize_caching.md

```diff
@@ -1,28 +1,50 @@
-This project involves improving coala's performance by implementing caching
-strategies. One example is to use git-like approach at file-collection level
-to not explore directories that haven't been modified. Caching per section can
-also be implemented so sections without issues don't rerun on all files where
-issues in other sections occurred.
+Currently, coala supports caching of project files. This was a big performance
+improvement over running code analysis for all files, even if most files
+are unchanged since the last run.
```

Comment:
The project should fix a problem. This probably is "but coala still is rather slow and could be a lot faster"